### PR TITLE
Add attribute which is used by ancestor classes to indicate the usage…

### PR DESCRIPTION
**Pull request by JorMaister**
… of tomo options

### CODE REVIEW

**Review by DStrelak — COMMENTED** [review]
are you sure that this is a correct file? 
Because it's flexalign for tomo version, so it makes sense it will be ued for tomography

**Comment by JorMaister** [comment]
Hi, @DStrelak ,
This is the best way I found to make other protocols involved, such as xmipp3 --> /protocol_movie_correlation.py, be aware of they are working with tomography, when it's the case. Thus, main protocol being executed is the one which has to indicate it to the others called. But, there can be better ways to do it, for sure. Any suggestions are more than welcome!

File: xmipptomo/protocols/protocol_flexalign.py [hunk 1]
```diff
@@ -1,53 +1,67 @@
...
+        # Get actual user decision about keeping aligned movies
+        doSaveMovie = self.doSaveMovie.get()
+
+        # If need to do even/odd
+        if self._doSplitEvenOdd():
+            self.doSaveMovie.set(True)
+
+        self._processMovie(tiltImageM)
+
+        import os
```
Comment (pconesa) [inline xmipptomo/protocols/protocol_flexalign.py:61]: COuld you please remove this line! I know is mine! :man_facepalming:

**Review by pconesa — CHANGES_REQUESTED** [review]

**Review by pconesa — APPROVED** [review]

File: xmipptomo/protocols/protocol_flexalign.py (hunk 1 above)
Comment (pconesa) [reply xmipptomo/protocols/protocol_flexalign.py]: I did it already.

**Pull request merged**